components/Table/Table.vue

```diff
@@ -26,13 +37,40 @@ export default {
     items: {
       type: Array,
       required: true
+    },
+    showExpand: Boolean
```

Comment:
The name is not so clear, it looks like it will show something more to the table. I suggest something like `expandable`.
Also please use the object definition to be strict and consistent with the rest of the code when defining the props.
Here is could be `{ type: Boolean, default: false }`

Reply:
Done.